tests/integration/test_concept.py

```diff
@@ -163,6 +163,8 @@ def test_set_label(self):
             # set label
             car_type = self.tx.get_schema_concept("car")
             car_type.label("vehicle")
+            # retrieve updated schema concepts
```

Comment:
why is this needed @vmax?

Reply:
@flyingsilverfin
```
q = tx.put_entity_type('q')
tx.get_schema_concept('q') # works
q.label('w')

tx.get_schema_concept('q') # no longer works (expected)
tx.get_schema_concept('w') # ALSO doesn't work
```

however:

```
tx.get_schema_concept('w')  # None
list(tx.query('match $x sub thing; get;')) # list<conceptmap>
tx.get_schema_concept('w') # it is there!
```

which leads me to believe schema concepts might be cached somewhere